guide/english/certifications/javascript-algorithms-and-data-structures/basic-algorithm-scripting/finders-keepers/index.md

```diff
@@ -55,3 +61,27 @@ function findElement(arr, func) {
 2. Use the function in the 2nd parameter as the callback function in arr.map()
 3. Acquire the index of the first number that meets the condition in the function.
 4. Use that index to display the first available number that meets the condition.
+
+## ![:rotating_light:](https://forum.freecodecamp.com/images/emoji/emoji_one/rotating_light.png?v=3 ":rotating_light:") Advanced Code Solution 2:
+```javascript
+function findElement(arr, func) {
+  return arr.find(func);
+}
+```
+![:rocket:](https://forum.freecodecamp.com/images/emoji/emoji_one/rocket.png?v=3 ":rocket:") <a href='https://repl.it/repls/FairSimultaneousBoolean' target='_blank' rel='nofollow'>Run Code</a>
+
+#### Relevant Links
+
+*   <a href='https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find' target='_blank' rel='nofollow'>Array.prototype.find</a>
```

Comment:
We prefer you to use markdown styling for the links.  See the [styling guide](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/style-guide-for-guide-articles.md#links)